hibernate-core/src/main/java/org/hibernate/boot/model/internal/CollectionPropertyHolder.java

```diff
@@ -326,6 +340,10 @@ public String toString() {
 	boolean prepared;
 
 	public void prepare(XProperty collectionProperty) {
+		prepare(collectionProperty, true);
```

Comment:
This default signature for `prepare` might not be necessart since `CollectionPropertyHolder` is in an internal package.

Reply:
Correct, you can just change the method. No need to care about backwards compatibility here.